db/seeds.rb

```diff
@@ -1,11 +1,30 @@
 Reservation.destroy_all
 Place.destroy_all
 User.destroy_all
-
-
 user = User.create!(name: 'John')
-place = Place.create!(description: 'Stay with Alvaro - Hosting for 6 years', photo: 'https://unsplash.com/photos/IWfe63thJxk', location: 'Stowe, Vermont', owner: user, rate: 123)
-second_place = Place.create!(description: 'Stay with Daniele - Consultancy', photo: 'https://unsplash.com/photos/1uNh3B3ppl4', location: 'Boston, MA, USA', owner: user, rate: 28)
-third_place = Place.create!(description: 'Stay with Mathieu - Business owner', photo: 'https://unsplash.com/photos/wE7qDv11kio', location: 'Gold Coast QLD, Australia', owner: user, rate: 55)
-fourth_place = Place.create!(description: 'Stay with David - Landscaper - Hosting for 7 years', photo: 'https://unsplash.com/photos/-4eDIVBRP2g', location: 'Juniper Mountain House, Evergreen', owner: user, rate: 80)
-Reservation.create!(start_date: '31/12/2022', end_date: '31/12/2023', place: place, customer: user, bill: 200)
\ No newline at end of file
+Place.create([
+    {description: "Stay with Alvaro - Hosting for 6 years",
+    photo: "https://images.unsplash.com/photo-1591825729269-caeb344f6df2?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=1470&q=80",
+    location: "Stowe, Vermont",
+    owner: user,
+    rate: 123
+    },
+    {description: "Stay with Daniele - Consultancy",
+    photo: "https://images.unsplash.com/photo-1619292585355-ab0e3fd509fe?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=872&q=80",
+    location: "Boston, MA, USA",
+    owner: user,
+    rate: 28
+    },
+    {description: "Stay with Mathieu - Business owner",
+    photo: "https://images.unsplash.com/photo-1629078691977-dc51747c0263?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=869&q=80",
+    location: "Gold Coast QLD, Australia",
+    owner: user,
+    rate: 55
+    },
+    {description: "Stay with David - Landscaper - Hosting for 7 years",
+    photo: "https://images.unsplash.com/photo-1634849430862-a5a3ba00a7f9?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=870&q=80",
+    location: "Juniper Mountain House, Evergreen",
+    owner: user,
+    rate: 80
+    },
+])
```

Comment:
[OPTIONAL] Could you please add some reservations to the seed file to test the MyReservations page functionality.

Reply:
Sure!